rest/config.go

```diff
@@ -61,43 +62,46 @@ const (
 
 	// Default number of index replicas
 	DefaultNumIndexReplicas = uint(1)
+
+	DefaultActiveReplicatorPingInterval = time.Minute * 5
```

Comment:
This seems to duplicate the defaultWebsocketPingInterval definition in active_replicator_config, which is a bit confusing - modifying the default there doesn't really have any effect from a SG perspective.  Seems reasonable to make it public and reference it directly?